server/odc-server/src/main/java/com/oceanbase/odc/server/web/controller/v2/DBRecyclebinController.java

```diff
@@ -54,47 +56,54 @@ public class DBRecyclebinController {
     public SuccessResponse<List<DBRecycleObject>> list(@PathVariable String sid) {
         return Responses.ok(recyclebinService.list(sessionService.nullSafeGet(SidUtils.getSessionId(sid), true)));
     }
 
     @ApiOperation(value = "purgeObject", notes = "purge a specific db object")
     @RequestMapping(value = "/purge/{sid:.*}", method = RequestMethod.POST)
+    @StatefulRoute(stateName = StateName.DB_SESSION, stateIdExpression = "#sid")
     public SuccessResponse<Boolean> purgeObject(@PathVariable String sid, @RequestBody List<DBRecycleObject> resource) {
         recyclebinService.purgeObject(sessionService.nullSafeGet(SidUtils.getSessionId(sid), true), resource);
         return Responses.ok(Boolean.TRUE);
     }
 
     @ApiOperation(value = "purgeAllObjects", notes = "purge all specific db objects")
     @RequestMapping(value = "/purgeAll/{sid:.*}", method = RequestMethod.POST)
+    @StatefulRoute(stateName = StateName.DB_SESSION, stateIdExpression = "#sid")
     public SuccessResponse<Boolean> purgeAllObjects(@PathVariable String sid) {
         recyclebinService.purgeAllObjects(sessionService.nullSafeGet(SidUtils.getSessionId(sid), true));
         return Responses.ok(Boolean.TRUE);
     }
 
     @ApiOperation(value = "flashback", notes = "flaskback db objects")
     @RequestMapping(value = "/flashback/{sid:.*}", method = RequestMethod.POST)
+    @StatefulRoute(stateName = StateName.DB_SESSION, stateIdExpression = "#sid")
     public SuccessResponse<Boolean> flashback(@PathVariable String sid, @RequestBody List<DBRecycleObject> resource) {
         recyclebinService.flashback(sessionService.nullSafeGet(SidUtils.getSessionId(sid), true), resource);
         return Responses.ok(Boolean.TRUE);
     }
 
     @ApiOperation(value = "getExpireTime", notes = "查看回收站对象过期时间，sid示例：sid:1000-1:d:db1")
     @RequestMapping(value = "/getExpireTime/{sid}", method = RequestMethod.GET)
+    @StatefulRoute(stateName = StateName.DB_SESSION, stateIdExpression = "#sid")
     public SuccessResponse<String> getExpireTime(@PathVariable String sid) {
         return Responses.ok(recyclebinSettingsService.getExpireTime(
                 sessionService.nullSafeGet(SidUtils.getSessionId(sid), true)));
     }
 
     @ApiOperation(value = "getRecyclebinSettings", notes = "查看回收站设置，sid示例：sid:1000-1:d:db1")
     @RequestMapping(value = "/settings/{sid}", method = RequestMethod.GET)
+    @StatefulRoute(stateName = StateName.DB_SESSION, stateIdExpression = "#sid")
     public SuccessResponse<RecyclebinSettings> getRecyclebinSettings(@PathVariable String sid) {
         return Responses.ok(recyclebinSettingsService.get(
                 sessionService.nullSafeGet(SidUtils.getSessionId(sid), true)));
     }
 
     @ApiOperation(value = "updateRecyclebinSettings", notes = "更新回收站设置，sid示例：sid:1000-1:d:db1")
     @RequestMapping(value = "/settings", method = RequestMethod.PATCH)
+    @StatefulRoute(stateName = StateName.DB_SESSION, stateIdExpression = "#sid")
```

Comment:
My oversight！The `updateRecyclebinSettings` api has stateId in its request body, and it may contain multiple `StatefulRoute`.
Here's how to fix it：
* Add a `multiState` parameter to the `@StatefulRoute` annotation, which is used to indicate that the current API may contain multiple state routings. Upon receiving this parameter, StatefulRoute will send requests to all routings.
* Add three default methods to the StateManager interface
  * `supportMultiRoute()` : Indicates that the current StateManager supports multiple states.
  * `Set<RouteInfo> getAllRoutes(Object stateId)` : Obtain all `StatefulRoute` based on stateId.
  * `Object handleMultiResponse(List<SingleNodeStateResponse> otherNodeResponse, Object currentNodeResult)`: Merge multiple response and reture a final response;